source/mainnet/net/release-notes/release-notes-lp.rst

```diff
@@ -1315,22 +1367,40 @@ Tools
 Concordium Client
 -----------------
 
-    August 9, 2023
-
-        Concordium Client 6.0.1 adds support for the upcoming `protocol version 6 <https://github.com/Concordium/concordium-update-proposals/blob/main/updates/P6.txt>`_ which is planned for release on Testnet August 21, 2023. **Note that this version of Concordium-Client requires at least node version 5.4.**
+    October 10, 2023
 
-        - Removed a stray CTrue in output of ``consensus show-chain-parameters``.
+        Concordium Client 6.1.0 includes support for the following:
 
-        - Added ``raw GetNextUpdateSequenceNumbers`` subcommand.
+            - Added baker win-time command for determining the earliest time a specified baker is expected to bake.
 
-        - Added node version to the output of ``raw GetNodeInfo``.
+            - Stream consumption ends early if an error is returned.
```

Comment:
I don't think we need to include this here. I don't think it was observable in the old client.